plaso/cli/image_export_tool.py

```diff
@@ -807,14 +809,22 @@ def ProcessSources(self):
 
     if self._artifact_filters or self._filter_file:
       self._ExtractWithFilter(
-          self._source_path_specs, self._destination_path, self._output_writer,
-          self._artifact_filters, self._filter_file,
+          self._source_path_specs, self._destination_path,
+          self._output_writer, self._artifact_filters, self._filter_file,
           self._artifact_definitions_path, self._custom_artifacts_path,
           skip_duplicates=self._skip_duplicates)
     else:
       self._Extract(
-          self._source_path_specs, self._destination_path, self._output_writer,
-          skip_duplicates=self._skip_duplicates)
+          self._source_path_specs, self._destination_path,
+          self._output_writer, skip_duplicates=self._skip_duplicates)
+
+    json_data = []
+
+    with open(os.path.join(
+        self._destination_path, 'hashes.json'), 'w') as write_file:
```

Comment:
Please move the filename out to a class constant - _HASHES_FILENAME or similar.

Reply:
Done.